rasterio/_base.pxd

```diff
@@ -20,7 +20,7 @@ cdef class DatasetReader:
     cdef public object _block_shapes
     cdef public object _nodatavals
     cdef public object _read
-    cdef object env
+    cdef object _env
```

Comment:
@sgillies Can we make this a public variable with Python's `_private` nomenclature so we can access it for debugging?